src/app/layout.tsx

```diff
@@ -28,23 +28,23 @@ export default function RootLayout({
         <ThemeProvider>
         <nav>
               <div className='nav-left'>
-                <Link href="/" className='link'>
+              <Link href="https://marble.software/" className='link'>
                   <img src="internal/amber.svg" alt="amber" className='logo'/>
-                  <span className='title'>
-                    Amber
-                  </span>
                 </Link>
+              <span className='title'>
+                <Link href="/">
+                  Amber
+                </Link>
+              </span>
               </div>
                 <SearchBar/>
               <div className='nav-right'>
                 <Button>
                 <Icon src='internal/side-bar.svg' size='2rem'/>
```

Comment:
@Weemizo Can you indent this icon and all other icons in this page please?

```suggestion
                  <Icon src='internal/side-bar.svg' size='2rem'/>
```

Reply:
Sure thing